### PR TITLE
Package par_incr.0.1

### PR DESCRIPTION
**Pull request by dipeshkaphle**
### `par_incr.0.1`
Parallel Self Adjusting Computation
A library for incremental computation, with support for parallelism

---
* Homepage: https://github.com/ocaml-multicore/par_incr
* Source repo: git+https://github.com/ocaml-multicore/par_incr.git
* Bug tracker: https://github.com/ocaml-multicore/par_incr/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/par_incr/par_incr.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+maintainer: ["Dipesh Kafle"]
+authors: ["Dipesh Kafle"]
+license: "MIT"
+tags: ["ocaml" "incremental" "multicore-ocaml"]
+homepage: "https://github.com/ocaml-multicore/par_incr"
+doc: "https://ocaml-multicore.github.io/par_incr/par_incr/index.html"
+bug-reports: "https://github.com/ocaml-multicore/par_incr/issues"
+depends: [
+  "ocaml" {>= "5.0"}
+  "dune" {>= "3.6" & >= "3.6"}
```
Comment (kit-ty-kate) [inline packages/par_incr/par_incr.0.1/opam:15]:
```suggestion
  "dune" {>= "3.6"}
```

File: packages/par_incr/par_incr.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+doc: "https://ocaml-multicore.github.io/par_incr/par_incr/index.html"
+bug-reports: "https://github.com/ocaml-multicore/par_incr/issues"
+depends: [
+  "ocaml" {>= "5.0"}
+  "dune" {>= "3.6"}
+  "domainslib" {>= "0.5.0" & with-test}
+  "alcotest" {>= "1.7.0" & with-test}
+  "mdx" {>= "2.3.0" & with-test}
+  "current_incr" {>= "0.6.1" & with-test}
+  "incremental" {>= "0.15.0" & with-test}
```
Comment (kit-ty-kate) [inline packages/par_incr/par_incr.0.1/opam:20]:
```suggestion
  "incremental" {>= "v0.15.0" & with-test}
```

**Comment by dipeshkaphle** [comment]
Thanks a lot for the fixes and suggestions. I've added the changes to main(https://github.com/ocaml-multicore/par_incr).

**Comment by kit-ty-kate** [comment]
The missing `v` prefix for `incremental` is still missing in `main`

**Comment by dipeshkaphle** [comment]
Aah, sorry about that. Done now. Thanks a lot.

**Comment by mseri** [comment]
Thanks

**Pull request merged**